libs/features/mas/web-components/src/aem-fragment.js

```diff
@@ -6,7 +6,7 @@ sheet.replaceSync(':host { display: contents; }');
 
 const baseUrl =
     document.querySelector('meta[name="aem-base-url"]')?.content ??
-    'https://publish-p22655-e155390.adobeaemcloud.com';
+    'https://odin.adobe.com/';
```

Comment:
this extra slash at the end is causing troubles with double slash in the fragment url.

https://odin.adobe.com//adobe/sites/cf/fragments/3d26df5b-0784-4967-8149-8a9e59131084

Reply:
oh thx for catching it, fixed